frontend/src/views/pages/inventory/InventorySlot.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <td v-html="value"></td>
```

Comment:
SonarCloud flagged this line as a security hotspot because you are using the `v-html` prop. A way to solve this would be by putting the html value directly in the `<td></td>` tags like this: `<td>{{ value }}</td>`

Reply:
This is address in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/52/commits/6d067bd2b87c4a5d34eede74a10719c101e6f202)